api/protosigverifierservice/verifier.proto

```diff
@@ -11,32 +11,30 @@ option go_package = "github.com/hyperledger/fabric-x-committer/api/protosigverif
 package protosigverifierservice;
 
 import "api/protoblocktx/block_tx.proto";
+import "api/protocoordinatorservice/coordinator.proto";
 
 service Verifier {
-    rpc StartStream(stream RequestBatch) returns (stream ResponseBatch);
+    rpc StartStream(stream Batch) returns (stream ResponseBatch);
 }
 
-message RequestBatch {
+message Batch {
     optional Update update = 1;
-    repeated Request requests = 2;
+    repeated Tx requests = 2;
 }
 
 message Update {
     optional protoblocktx.ConfigTransaction config = 1;
     optional protoblocktx.NamespacePolicies namespace_policies = 2;
 }
 
-message Request {
-    uint64 block_num  = 1;
-    uint64 tx_num = 2;
-    protoblocktx.Tx tx = 3;
+message Tx {
+    protocoordinatorservice.TxId id = 1;
+    protoblocktx.Tx tx = 2;
```

Comment:
I assume we use the txID from the `id` and the whole `tx` for signature verification.